### PR TITLE
Mark more user-visible strings as translatable

### PR DESCRIPTION
**Pull request by hughsie**
Fixes https://github.com/fwupd/fwupd/issues/2616

Type of pull request:
- [ ] New plugin (Please include [new plugin checklist](https://github.com/fwupd/fwupd/wiki/New-plugin-checklist))
- [X] Code fix
- [ ] Feature
- [ ] Documentation

### CODE REVIEW

**Comment by hughsie** [comment]
@bellaperez is that all of them?

**Comment by bellaperez** [comment]
Fine. Thank you ;-)

File: plugins/dfu/dfu-tool.c [hunk 1]
```diff
@@ -997,13 +997,15 @@ main (int argc, char *argv[])
...
+		     /* TRANSLATORS: command argument: uppercase, spaces->dashes */
+		     _("FILENAME"),
 		     /* TRANSLATORS: command description */
 		     _("Read firmware from device into a file"),
 		     dfu_tool_read);
 	dfu_tool_add (priv->cmd_array,
 		     "read-alt",
-		     "FILENAME DEVICE-ALT-NAME|DEVICE-ALT-ID",
+		     /* TRANSLATORS: command argument: uppercase, spaces->dashes */
+		     _("FILENAME DEVICE-ALT-NAME|DEVICE-ALT-ID"),
```
Comment (superm1) [inline plugins/dfu/dfu-tool.c:1008]: Do we need to worry about translators making these into multiple words on translation and then the number of arguments gets confused?

**Review by superm1 — APPROVED** [review]
Just a question on the comments around the translated words and potential impact from that.

**Comment by bellaperez** [comment]
> Just a question on the comments around the translated words and potential impact from that.

Very useful! For Catalan language I made:
  FILENAME -> NOM_FITXER

File: plugins/dfu/dfu-tool.c (hunk 1 above)
Comment (hughsie) [reply plugins/dfu/dfu-tool.c:1008]: I put a note in the translator comment to that effect.

**Pull request merged**